File: Models/BiomaSurfaceSWATSoilSWATC/src/cpp/BiomaSurfaceSWATSoilSWATC/SurfaceSWATSoilSWATCAuxiliary.cpp
#include "SurfaceSWATSoilSWATCAuxiliary.h"
using namespace BiomaSurfaceSWATSoilSWATC;


SurfaceSWATSoilSWATCAuxiliary::SurfaceSWATSoilSWATCAuxiliary() {}

double SurfaceSWATSoilSWATCAuxiliary::getAboveGroundBiomass() { return this->AboveGroundBiomass; }
double SurfaceSWATSoilSWATCAuxiliary::getSurfaceSoilTemperature() { return this->SurfaceSoilTemperature; }

void SurfaceSWATSoilSWATCAuxiliary::setAboveGroundBiomass(double _AboveGroundBiomass) { this->AboveGroundBiomass = _AboveGroundBiomass; }
void SurfaceSWATSoilSWATCAuxiliary::setSurfaceSoilTemperature(double _SurfaceSoilTemperature) { this->SurfaceSoilTemperature = _SurfaceSoilTemperature; }

File: Models/BiomaSurfaceSWATSoilSWATC/src/java/BiomaSurfaceSWATSoilSWATC/SurfaceSWATSoilSWATCAuxiliary.java
import  java.io.*;
import  java.util.*;
import java.time.LocalDateTime;

public class SurfaceSWATSoilSWATCAuxiliary
{
    private double AboveGroundBiomass;
    private double SurfaceSoilTemperature;
    
    public SurfaceSWATSoilSWATCAuxiliary() { }
    
    public SurfaceSWATSoilSWATCAuxiliary(SurfaceSWATSoilSWATCAuxiliary toCopy, boolean copyAll) // copy constructor 
    {
        if (copyAll)
        {
            this.AboveGroundBiomass = toCopy.getAboveGroundBiomass();
            this.SurfaceSoilTemperature = toCopy.getSurfaceSoilTemperature();
            this.SurfaceSoilTemperature = toCopy.getSurfaceSoilTemperature();
        }
    }
    public double getAboveGroundBiomass()
    { return AboveGroundBiomass; }

    public void setAboveGroundBiomass(double _AboveGroundBiomass)
    { this.AboveGroundBiomass= _AboveGroundBiomass; } 
    
    public double getSurfaceSoilTemperature()
    { return SurfaceSoilTemperature; }

    public void setSurfaceSoilTemperature(double _SurfaceSoilTemperature)
    { this.SurfaceSoilTemperature= _SurfaceSoilTemperature; } 
    
}